cv_next/components/ui/Carousel/carouselArrows.tsx

```diff
@@ -1,10 +1,5 @@
 "use client";
-import React, {
-  ComponentPropsWithRef,
-  useCallback,
-  useEffect,
-  useState,
-} from "react";
+import { ComponentPropsWithRef, useCallback, useEffect, useState } from "react";
```

Comment:
Is this after formatting? iirc the formatting is like the previous version, not this one

Reply:
I think that it's because I deleted the React import(it was the formatter tho)

Reply:
It's based on the length(when I've added another useSomething it was vertical)